packages/experimental/Drawer/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui-react-native/experimental-drawer",
+  "version": "0.0.1",
+  "description": "A Native Drawer component using the Fluent Design System. Currently only implemented on Android",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/component-cache": "^1.3.0",
+    "@fluentui-react-native/framework": "0.7.0",
+    "tslib": "^1.13.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "immediate": "~3.0.5",
+    "lie" : "~3.3.0",
+    "pako" : "~1.0.2",
+    "react": "16.13.1",
+    "react-native": "^0.63.4",
```

Comment:
We should bump this to the same version of react=native we're using for peer dependencies.
Same for the react version